desktop/release-notes.md

```diff
@@ -24,11 +24,63 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.21.0
+
+{% include release-date.html date="2023-06-29" %}
+
+{% include desktop-install.md all=true version="4.21.0" build_path="/" %}
+
+### New
+
+- Added support for new Wasm runtimes: slight, spin, and wasmtime.  Users can download Wasm runtimes on demand when the containerd image store is enabled.
+- Added Rust server support to Docker init.
+- New [Builds view](use-desktop/builds.md) that lets you inspect builds and manage builders.
```

Comment:
@dvdksn I forgot about this but maybe we should say that it's an experimental feature and can be enabled in "Features in development" in Docker Desktop settings?

Reply:
Yeah we could just add "(beta)" after the link I think.

Reply:
How about? 
```suggestion
- New [Builds view](use-desktop/builds.md) (Beta) that lets you inspect builds and manage builders. This can be found in the **Features in Development** tab in Docker Desktop.
```

Reply:
Updated the PR!